aten/src/ATen/nnapi/nnapi_bind.h

```diff
@@ -36,7 +36,19 @@ struct NnapiCompilation : torch::jit::CustomClassHolder {
     NnapiCompilation() = default;
     ~NnapiCompilation() override = default;
 
-    TORCH_API void init(at::Tensor serialized_model_tensor, std::vector<at::Tensor> parameter_buffers);
+    TORCH_API void init(
+      at::Tensor serialized_model_tensor,
+      std::vector<at::Tensor> parameter_buffers
+    );
+
+    TORCH_API void init_with_preferences(
```

Comment:
I might just call this "init2" so we can create an "init3" if we need to add more args in the future.  Feels a bit hacky, but I've seen it done successfully before: https://subversion.apache.org/docs/api/latest/group__svn__io__byte__streams.html#ga444db383401ed714a240ff6ca5ca76b3 (svn_stream_copy).